front/poke/temporal/activities.ts

```diff
@@ -527,6 +528,11 @@ export async function deleteSpacesActivity({
   });
 
   for (const space of spaces) {
+    const res = await space.delete(auth, { hardDelete: false });
```

Comment:
Why did we want to enforce that it must be soft deleted first? (I see the isDeletable from scrubSpaceActivity checking that it's either soft deleted or system/global/conversation)

Reply:
Yep so non system/global/conversation must be soft deleted first :+1:

Reply:
Then we should exclude them from here, no? Reading the code we fetch all workspace's spaces then soft delete all of them.

Reply:
Yep the scrubSpaceActivity checks canDelete which require softDeletion so we need to softDelete them before calling it. Does that make sense?

I'm not sure why we enforce this canDelete, really just complied with it here.

Reply:
I presume you mean `isDeletable`, right? If so, this method already handles all default spaces, so they don't need to be soft deleted first. This let you have a proper authenticator until the very end which you won't get if the default spaces (mainly global) have already been deleted.

Reply:
Yes the problem are the non default spaces :+1: (and soft deleting the default one is not really a problem here)